src/System.IO.FileSystem/tests/GetTime_str.cs

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Collections;
+using System.Globalization;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Runtime.CompilerServices;
+using Xunit;
+
+namespace System.IO.FileSystem.Tests
+{
+    public abstract class GetTime_str : FileSystemTest
+    {
+        private readonly string NonExistantFileName = "nonexistant_file_name";
+        private readonly TimeSpan CreateCheckGap = TimeSpan.FromMinutes(8);
+
+        protected abstract DateTime GetTime(string path);
+
+        [Fact]
+        public void FileNotFound()
+        {
+            // If the directory described in the path parameter does not exist,
+            // this method returns 12:00 midnight, January 1, 1601 A.D. (C.E.)
+            // Coordinated Universal Time (UTC), adjusted to local time.
+            DateTime time = GetTime(NonExistantFileName);
+
+            Assert.Equal(time, new DateTime(1601, 1, 1, 0, 0, 0, DateTimeKind.Utc).ToLocalTime());
+        }
+
+        [Fact]
+        public void PathArgumentWhitespace()
+        {
+            // Check ArgumentException is thrown for only whitespace in path
+            Action<string> test = (s) => Assert.Throws<ArgumentException>(() => GetTime(s));
```

Comment:
Whitespace is valid in a filename on Unix (even if the whole filename is entirely whitespace).